example/src/screens/Room.tsx

```diff
@@ -119,15 +120,24 @@ export const Room = ({ navigation }: Props) => {
                   .map((p) => (
                     <View
                       key={p.id}
-                      style={getStylesForParticipants(participants)}
+                      style={[
+                        getStylesForParticipants(participants),
+                        styles.shownParticipantBorder,
+                      ]}
                     >
                       <RoomParticipant participant={p} />
                     </View>
                   ))}
 
                 {participants.length > MAX_NUM_OF_USERS_ON_THE_SCREEN && (
                   <View style={getStylesForParticipants(participants)}>
-                    <Typo variant="label">Others</Typo>
+                    <OtherParticipants
+                      p1={participants[participants.length - 1]}
+                      p2={participants[participants.length - 2]}
+                      numOfOtherParticipants={
+                        participants.length - MAX_NUM_OF_USERS_ON_THE_SCREEN
```

Comment:
Ok, one more bug: the number of other participants is invalid. For 10 participants there is no text (7 participants on screen, 2 circles so the text should say 1 other participant) and for more there is 1 participant missing.

Reply:
Good catch, I forgot that when others tab appears there's one less space for a regular participant. Should be fixed now.